### PR TITLE
Save TicketConference.assigned_to email always in lower case

### PR DESCRIPTION
**Pull request by cezar77**
fix issue #740

### CODE REVIEW

**Comment by artcz** [comment]
Hi Cezar!

Looks like you need to rebase your branch, because it has too many commits from other sources.

What you need to execute on your fork is:

`git fetch upstream`
and then follow it with
`git rebase upstream/dev/ep2019`

just like in step 3 here: https://github.com/EuroPython/epcon#contributing

Note: it may have some conflicts, so you might need to resolve them, but if you have any problems with that let me know.

**Comment by cezar77** [comment]
@artcz Thank you for the hints.

I rebased the branch. I hope it should work now.

**Review by matrixise — APPROVED** [review]

File: p3/tests/test_models.py [hunk 1]
```diff
@@ -43,6 +43,23 @@ def test_profile(self):
 
         self.assertEqual(attendee_profile, profile)
 
+    def test_assigned_to_email_is_saved_as_lowercase(self):
```
Comment (artcz) [inline p3/tests/test_models.py]: I'd like to move this test somewhere inside `tests/test_tickets.py` (ie. global test/ directory, not the per-app directory)

**Review by artcz — CHANGES_REQUESTED** [review]
I added a comment about test location, + we still need to get correct rebase in place.

File: p3/tests/test_models.py (hunk 1 above)
Comment (cezar77) [reply p3/tests/test_models.py]: Please apologise the very late reply. I moved the test to `tests/test_tickets.py`.

**Comment by aramboi** [comment]
This was done as part of PR #1173

**Pull request closed without merging**